src/EFCore.MySql.Json.Microsoft/Infrastructure/IMysqlJsonOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Pomelo Foundation. All rights reserved.
+// Licensed under the MIT. See LICENSE in the project root for license information.
+
+using System.Text.Json;
+
+namespace Pomelo.EntityFrameworkCore.MySql.Json.Microsoft.Infrastructure;
+
+public interface IMysqlJsonOptions
```

Comment:
This looks like a casing issue; shouldn't it be `IMySqlJsonOptions` for consistency?

Reply:
I will change it